### DISCUSSION TITLE
About advantages and disadvantages of youtube login

### DISCUSSION THREAD

**Discussion by rairay91** [InfinityLoop1308/PipePipe#512] [Q&A]
How well youtube login is going and what extra feature will i get? Will background playback stop functioning after login using youtube account?
In summary what are the advantages and disadvantages of youtube login?

**Comment by InfinityLoop1308** [comment]
https://github.com/InfinityLoop1308/PipePipe?tab=readme-ov-file#about-sign-in
> PipePipe will ONLY use the login cookie for the specified scenarios you set. You can configure it in "Cookie Functions."
> 
> For YouTube, the cookie will only be used when retrieving playback streams.

Advantage: 
- You can watch videos normally even if your ip is banned (The 'Sign in to confirm you are not a bot' error)
- You can watch age-restricted videos
- You can watch paid contents if your account paid (haven't tested but it should work)
- We may support fetch subscription from account in the future.

Disadvantage: None. Nothing will stop working. Nothing will be uploaded to cloud servers or any other places. The only difference is that YouTube will know "User xxxxx is requesting an API", that's all.
